projects/v2/src/app/app-config.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable }  from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { ReplaySubject } from 'rxjs';
+import { SheetService } from './services/sheet.service';
+import { SheetDetails } from './models/sheet.model';
+ 
+@Injectable()
+export class ConfigService {
+
+    config = new ReplaySubject(2);
+    sheetConfiguration = new ReplaySubject<SheetDetails[]>(2);
+ 
+    constructor(private readonly http: HttpClient,public sheetservice: SheetService) {
+    }
+    
+    Init() {
```

Comment:
Given above, you do not need this Init function at all (or the app initializer piece)

Reply:
Let me know if this change causes issue. It really shouldn't if you are using observables correctly.